.github/workflows/cron.yml

```diff
@@ -1,28 +1,73 @@
-name: cron
+name: Cron
```

Comment:
Oh, I didn't think you'd copy all the workflow into another file. I meant we could inject a small if-clause on the job level to filter out things.

Reply:
Could you post a suggestion?

Reply:
Something like this should do https://github.com/jazzband/pip-tools/pull/1012/files#r419714725

Reply:
I'm okay with the copy approach for a while. As soon as GitHub provides an opportunity to make matrix conditionals at a job level we'll refactor it.

Reply:
Fair enough.

Reply:
Well, there *is* a crazy way to implement this: https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#fromjson 🤪

Reply:
Wow 😄